### PR TITLE
Update Li's student status

### PR DESCRIPTION
**Pull request by gqlo**
update student status

### CODE REVIEW

**Comment by KoheiIchikawa** [comment]
You have overwritten your English profile with Japanese contents.
If you want to have both English and Japanese profiles, you first have to remove the symlink from Japanese profile to English profile.
```
cd content/ja/authors
rm guoqing-li
```
Then, copy the contents from English profile.
```
cp -a ../../en/authors/guoqing-li ./
```
And then, edit the following two profile files accordingly.
- content/en/authors/guoqing-li/_index.md
- content/ja/authors/guoqing-li/_index.md

**Comment by gqlo** [comment]
> cd content/ja/authors

thanks, that's exactly what i was looking for

File: content/en/authors/guoqing-li/_index.md [hunk 1]
```diff
@@ -10,28 +10,28 @@ authors:
 superuser: false
 
 # Role/position (e.g., Professor of Artificial Intelligence)
-role: Intern
+role: Master's Student
 
 # Organizations/Affiliations
 organizations:
-- name: Nara Institute of Science and Technology
+- name: Nara Institue of Science and Technology
```
Comment (KoheiIchikawa) [inline content/en/authors/guoqing-li/_index.md:17]: Institue -> Institute

File: content/ja/authors/guoqing-li/_index.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+# Is this the primary user of the site?
+superuser: false
+
+# Role/position (e.g., Professor of Artificial Intelligence)
+role: 博士前期課程
+
+# Organizations/Affiliations
+organizations:
+- name: 先端科学技術大学院大学
```
Comment (KoheiIchikawa) [inline content/ja/authors/guoqing-li/_index.md:17]: 先端科学技術大学院大学 -> 奈良先端科学技術大学院大学

**Review by KoheiIchikawa — APPROVED** [review]

**Pull request merged**